codegen/directive.go

```diff
@@ -3,9 +3,9 @@ package codegen
 import "strings"
 
 type Directive struct {
-	name string
+	Name string
 }
 
-func (d *Directive) Name() string {
-	return strings.Title(d.name)
+func (d *Directive) GoName() string {
```

Comment:
maybe this should be NameUpper? where is this used?

Reply:
also, https://github.com/vektah/gqlgen/blob/master/codegen/templates/templates.go#L65 is available in templates